rts/at/tx/src/main/java/org/jboss/jbossts/star/service/TMApplication.java

```diff
@@ -46,7 +46,17 @@ public class TMApplication extends Application {
     HashSet<Object> singletons = new HashSet<Object>();
     Set<Class<?>> classes = new HashSet<Class<?>> ();
 
+
     public TMApplication(Class<?> ... extraClasses) {
+        this();
+        try
```

Comment:
I'm pretty sure the style says we need a blank line before try, if not then the policy is that we should match the existing format in the file.

Reply:
I've just added the blank line before try. Thanks